examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -3109,7 +3109,9 @@ void pxScriptView::runScript()
     mReady = new rtPromise();
 #endif
 
-    mCtx->runFile("init.js");
+    string s = getenv("PXSCENE_PATH");
```

Comment:
Please rename "PXSCENE_PATH" to "SPARK_PATH".  Also, can you please use rtString instead of string?

Reply:
Please see the updates on this